tests/cases/trait/one-per-file

```diff
@@ -19,4 +19,5 @@ trait Bar
 
 ---MESSAGES---
 11:1 PSR1.Classes.ClassDeclaration.MultipleClasses
+11:1 Squiz.Classes.ClassFileName.NoMatch
```

Comment:
Hmm, this now requires PHP_CodeSniffer `^3.4`. Wonder if these message checks should check that they are present, not the exact match? Means not raising the constraint when we're not actually using new features.

Could add an ignore/possible messages section, that excludes messages if they're found.

Reply:
Opened https://github.com/libero/php-coding-standard/pull/37.